html/browsers/the-window-object/proxy-getOwnPropertyDescriptor.html

```diff
@@ -0,0 +1,95 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <meta charset="utf-8">
+  <title>getOwnPropertyDescriptor() is correct for Proxy with host object target</title>
+  <link rel="help" href="https://html.spec.whatwg.org/multipage/#window">
+  <link rel="help" href="https://heycam.github.io/webidl/#Unforgeable">
+  <script src="/resources/testharness.js"></script>
+  <script src="/resources/testharnessreport.js"></script>
+</head>
+<body>
+<script>
+'use strict';
+
+test(() => {
+  let windowProxy = new Proxy(window, {});
+  name = 'old_name';
+  let descriptor = Object.getOwnPropertyDescriptor(windowProxy, 'name');
+
+  assert_equals(descriptor.get.call(window), 'old_name');
+  descriptor.set.call(window, 'new_name');
+  assert_equals(name, 'new_name');
+  assert_true(descriptor.enumerable);
+  assert_true(descriptor.configurable);
+}, 'Window target, no trap, "name" attribute');
+
+test(() => {
+  let windowProxy = new Proxy(window, {});
+
+  assert_object_equals(
+    Object.getOwnPropertyDescriptor(windowProxy, 'document'),
+    Object.getOwnPropertyDescriptor(window, 'document')
+  );
+}, 'Window target, forwarding trap, [Unforgeable] "document" attribute');
```

Comment:
I'm not very familiar with proxies, why does this have a forwarding trap whereas the initial test does not?

Reply:
I was trying to cover all cases with as few test cases as possible.
While "custom trap" appears to be a common case, we really need "no trap" cases as runtimes used to incorrectly forward calls to Proxy targets.
For [`[Unforgeable]`](https://heycam.github.io/webidl/#Unforgeable) properties, that are non-configurable, trap results are very limited due to [Invariants of the Essential Internal Methods](https://tc39.es/ecma262/#sec-invariants-of-the-essential-internal-methods), so as of 5b78341 we just increment `trapCalls` and make sure `Proxy` doesn't throw.